projects/feature-activation/0005-miner-update-request.md

```diff
@@ -0,0 +1,61 @@
+# Hathor Integration Update Request
+
+## Description
+
+We’re currently in the testing phase of a project called Feature Activation, which is a mechanism used to activate new consensus-agreed features on Hathor Network. In the future, it will be used to release important features such as Nano Contracts and increasing the maximum Merkle Path length, which will allow miners to find more valid merge mined blocks. Here’s a [blog post](https://blog.hathor.network/hathors-feature-activation-process-miners-lead-the-way-d7f43b12978d) about it.
+
+In order for that to work, we need miners to propagate a support signal for features through their mined blocks. This is simply a repurposed byte in the block data. We’ll soon have a complete guide on how to interact with feature signals, but everything should work by default and mostly transparently. However, at this time, we kindly request your cooperation.
+
+It’s likely that you need to update your integration with Hathor to correctly propagate those signals. We have a new field called `signal_bits` in our block template, that you currently use. This field is an integer and represents a single byte. You simply have to set this as the first byte in the Hathor blocks you generate, and everything should work. Before, the `version` field occupied the first 2 bytes. Now, the `signal_bits` is the first byte, and the version field is the second byte.
+
+We of course are available for any questions or help that you might need. Thank you in advance!
+
+## Updating the Integration
+
+In order to support this new feature, a simple change must be made.
+
+### 1. Identifying where the Hathor block is generated
+
+Your current integration most certainly reads our Block template from one of our HTTP or WebSocket endpoints. The fields provided in those endpoints are the following:
```

Comment:
Can we be more specific here, by mentioning the specific API they use?

Reply:
@jansegre ?

Reply:
They could also be using the ``/v1a/get_block_template` HTTP API. So we might mention it in the guide.

Reply:
Added in https://github.com/HathorNetwork/rfcs/pull/74/commits/9cb72cc16011c3ff7ccdb4ab91a2b6518dbcee52

Reply:
I remember there being 3 different mining APIs, an older HTTP, a newer HTTP and a WebSocket, the newer HTTP and WebSocket should have the same fields (for the most part). I'll take a look to confirm this.